app/views/catalog/_show_digital_content.html.erb

```diff
@@ -1 +1,7 @@
+<!-- Viewer for the current MMS_ID -->
 <div id="view" class="document-viewers" data-bib-id="<%= @document.id %>"></div>
+
+<!-- Viewer for related MMS_IDs -->
+<% @document.related_bibs_iiif_manifest.each_with_index do |related_bib, ix| %>
```

Comment:
This new viewer will be picked up by Javascript in https://github.com/pulibrary/orangelight/blob/main/app/javascript/orangelight/orangelight_ui_loader.es6#L37-L43 because it has the `document-viewers` CSS class.